src/ssh-util/src/tunnel.rs

```diff
@@ -30,19 +31,36 @@ use crate::keys::SshKeyPair;
 // TODO(benesch): allow configuring the following connection parameters via
 // server configuration parameters.
 
-/// How often to check whether the SSH session is still alive.
-const CHECK_INTERVAL: Duration = Duration::from_secs(30);
+pub const DEFAULT_CHECK_INTERVAL: Duration = Duration::from_secs(30);
+pub const DEFAULT_CONNECT_TIMEOUT: Duration = Duration::from_secs(30);
+pub const DEFAULT_KEEPALIVES_IDLE: Duration = Duration::from_secs(10);
 
-/// The timeout to use when establishing the connection to the SSH server.
-const CONNECT_TIMEOUT: Duration = Duration::from_secs(30);
+/// Configuration of Ssh session and tunnel timeouts.
+// Its 3 `Duration`s, its fine to `Copy` it.
+#[derive(Copy, Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct SshTimeoutConfig {
+    /// How often to check whether the SSH session is still alive.
+    pub check_interval: Duration,
+    /// The timeout to use when establishing the connection to the SSH server.
+    pub connect_timeout: Duration,
+    /// The idle time after which the SSH control master process should send a
```

Comment:
I'm too woke––is there another phrase or term we can use instead of master that's still technically appropriate?

Reply:
I just moved this around but I agree! Ill change it to "leader"